blueprints/datamodel.tex

```diff
@@ -335,7 +335,117 @@ \subsection{Delivery Services}
 content and a set of Cache Servers and configuration options used to distribute
 that content.
 
-\subsubsection{Status}
+\subsubsection{Common Properties}
+Herein described are the properties common to all Delivery Service objects. The
+Routing Type of a Delivery Service encapsulates the methods by which clients
+may request content routing, and depending on its value the Delivery Service
+takes on a set of additional properties. Put simply, these are all different
+types of objects that are closely related.\\
+This section details all of the properties that are common to \emph{all} types of
+Delivery Services.
+
+\paragraph{Anonymous Blocking}
+A Delivery Service that has "Anonymous Blocking" tells Traffic Router to block
+requests from anonymized IP addresses. Whether or not and how well that can
+actually be done is dependent on the configuration of each Traffic Router itself,
+and if this Delivery Service is DNS-routed the only IP address Traffic Routers
+will be capable of checking for anonymization (e.g. known proxy/VPN/TOR exit node)
+then
```

Comment:
Note this is a global configuration underneath this boolean flag.  There's an open issue to allow two DS to vary in the configuration.

Reply:
So you're saying that anonymous blocking is configured on a Traffic Router today, and individual DS settings have no effect?

Reply:
You can't express different settings per DS, it's all or nothing per ds and one config per cdn.  https://github.com/apache/trafficcontrol/issues/4227

Reply:
Well you can't define subsets of anonymous blocking or IP whitelists on a per-Delivery Service basis, but as I understand that issue the simple "check for anonymization vs don't" flag does work as advertised. Correct?

Do you think Delivery Services also need a property to define IP whitelists for the purposes of anonymous blocking and/or geographic blocking? And maybe also a "policy" of what to consider "anonymized"? That second one sounds pretty hard to do without digging into TR implementation details.

Reply:
That's not how I've understood it.  Per the docs it's a TR profile setting which per CDN, and it's only a boolean to turn whatever that profile has enabled on or off.